### DISCUSSION TITLE
How can I get this achievement?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#18594] [Other Feature Feedback, Questions, & Ideas]
![image](https://user-images.githubusercontent.com/63979746/173617157-75a871a6-648f-4a4e-aae4-b96b8f709c33.png)

**Comment by herkura** [comment]
Hi @protyodev !
1) After answering two discussions, you will get the "Galaxy Brain" badge.
2) Get Two PR's merged, and get the "Pull Shark" badge.
3) For the "YOLO" badge, get your PR reviewed by a third party or a different collaborator in your repo.

Hope this helps 😄

**Reply by ghost** [reply]
Thanks

**Reply by herkura** [reply]
@protyodev If you found it helpful, kindly consider making it as an answer :)

**Reply by narmeranyo** [reply]
thanks

**Reply by WHAHA-HA** [reply]
if you give me a star, I will do it too.
https://github.com/WHAHA-HA/spring-boot-react-ecommerce
thank u

**Comment by emerisly** [comment]
for @herkura 's answer about "galaxy brain", you also need to be marked as an answer by others

**Reply by emerisly** [reply]
you might found this [repo](https://github.com/Schweinepriester/github-profile-achievements) helpful

**Comment by Marck** [comment]
And for the Shark badge, merging PRs on your own repositories should work too

_If you find this answer usefull, consider marking it as an answer_

**Reply by WHAHA-HA** [reply]
if you give me a star, I will do it too.
https://github.com/WHAHA-HA/spring-boot-react-ecommerce
thank u

**Comment by 5HT2** [comment]
This repo has more info: https://github.com/Schweinepriester/github-profile-achievements/

**Reply by WHAHA-HA** [reply]
if you give me a star, I will do it too.
https://github.com/WHAHA-HA/spring-boot-react-ecommerce
thank u

**Comment by elsowiny** [comment]
Lol @protyodev
![giphy](https://user-images.githubusercontent.com/6583011/173967239-e10bafd0-d9d5-4d06-a6d4-c56169126282.gif)

**Comment by submgr** [comment]
Is it something limited? I'm about this *Send feedback* which currently in beta. Cuz it's not that much people here comparing with whole Github community:)

**Reply by WHAHA-HA** [reply]
if you give me a star, I will do it too.
https://github.com/WHAHA-HA/spring-boot-react-ecommerce
thank u

**Comment by dusens** [comment]
amz

**Reply by WHAHA-HA** [reply]
if you give me a star, I will do it too.
https://github.com/WHAHA-HA/spring-boot-react-ecommerce
thank u
